requirements/base.in

```diff
@@ -3,7 +3,7 @@
 -c constraints.txt
 
 click>=7.0,<8.0
-Django>=1.11,<2.0          # Web application framework
```

Comment:
Are we leaving Django 1.11 in here for reasons? Or can we simply just make it `>=2.2,<2.3` or something along those lines? If we are leaving it for now, do we have plans to remove it at some point?

Reply:
I left it because I assumed that some repos that are still on python2 will still need to use this code.